### PR TITLE
Improve Django Admin integration with original package

### PR DESCRIPTION
**Pull request by NikAzanov**
Prior to this, all admin models and admin templates were rewritten from the original package and didn't use inheritance. In this PR I tried to reuse the `django-import-export` code by inheritance to reduce the amount of code and make it easier to make changes.

In this PR:
* Improve GitHub workflow name
* Get rid of `DjangoObjectActions` and implement default django admin action instead (Maybe later we can extend this)
* Use `mixins.BaseExportMixin`, `mixins.BaseImportMixin` and `admin.ImportExportMixinBase` from original package for celery admin mixins
* Use `admin/import_export/` templates instead of copies in `admin/import_export_ext/`
* Small improvements:
	* Fix static folder name
	* Fix invoke command to run celery
	* Fix progress bar widget
	* Rename `filter_class` to `filterset_class`
	* Add `cancel_job` action for exporting

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+name: Django-import-export-extensions CI
```
Comment (ron8mcr) [inline .github/workflows/checks.yml:1]: Let's find better name, actually this actions perform checks, and `CI` is too common name

File: import_export_extensions/admin/mixins/export_mixin.py [hunk 2]
```diff
@@ -32,67 +42,30 @@ class CeleryExportAdminMixin:
...
-
     # template used to display ExportForm
-    celery_export_template_name = "admin/celery_export.html"
+    celery_export_template_name = "admin/import_export/export.html"
 
-    export_status_template_name = "admin/celery_export_status.html"
+    export_status_template_name = "admin/import_export_ext/celery_export_status.html"
 
-    export_results_template_name = "admin/celery_export_results.html"
+    export_results_template_name = "admin/import_export_ext/celery_export_results.html"
```
Comment (ron8mcr) [inline import_export_extensions/admin/mixins/export_mixin.py:53]: Let's use full name "import_export_extensions", just like in `INSTALLED_APPS`

File: import_export_extensions/admin/model_admins/export_job_admin.py [hunk 3]
```diff
@@ -211,5 +214,19 @@ def get_fieldsets(
...
 
+    @admin.action(description="Cancel selected jobs")
+    def cancel_jobs(self, request, queryset):
+        """Admin action for cancelling data export."""
+        for obj in queryset:
+            try:
+                obj.cancel_export()
+                self.message_user(
+                    request,
+                    _("Export Canceled"),
```
Comment (ron8mcr) [inline import_export_extensions/admin/model_admins/export_job_admin.py:225]: Let's also add representation of failed `obj` to messages, ex:
"Export of {str(obj)} Canceled"

File: import_export_extensions/models/export_job.py [hunk 4]
```diff
@@ -298,6 +304,29 @@ def export_data(self):
...
 
+    def cancel_export(self) -> None:
+        """Cancel current data export.
+
+        ExportJob can be CANCELLED only from following states:
+            - CREATED
+            - EXPORTING
+
+        """
+        status_task_field_map = {
```
Comment (ron8mcr) [inline import_export_extensions/models/export_job.py:315]: Do you really need this map?

**Review by ron8mcr — CHANGES_REQUESTED** [review]

File: import_export_extensions/admin/mixins/import_mixin.py [hunk 5]
```diff
@@ -396,3 +371,23 @@ def _redirect_to_results_page(
                 cache.delete(key)
 
         return HttpResponseRedirect(redirect_to=url)
+
+    def has_import_permission(self, request):
```
Comment (yalef) [inline import_export_extensions/admin/mixins/import_mixin.py:375]: Let's add typing for `request`

File: import_export_extensions/admin/model_admins/export_job_admin.py [hunk 6]
```diff
@@ -36,6 +36,9 @@ class ExportJobAdmin(
     export_job_model = models.ExportJob
     list_filter = ("export_status",)
     list_select_related = ("created_by",)
+    actions = [
```
Comment (yalef) [inline import_export_extensions/admin/model_admins/export_job_admin.py:39]: can we use tuple here?

File: import_export_extensions/admin/model_admins/export_job_admin.py [hunk 7]
```diff
@@ -211,5 +214,19 @@ def get_fieldsets(
 
         return [status, traceback_fields, export_params]
 
+    @admin.action(description="Cancel selected jobs")
+    def cancel_jobs(self, request, queryset):
```
Comment (yalef) [inline import_export_extensions/admin/model_admins/export_job_admin.py:218]: let's add typings

File: import_export_extensions/admin/model_admins/import_job_admin.py [hunk 8]
```diff
@@ -39,9 +38,10 @@ class ImportJobAdmin(
     import_job_model = models.ImportJob
     list_filter = ("import_status",)
     list_select_related = ("created_by",)
-
-    class Media:
-        pass
+    actions = [
```
Comment (yalef) [inline import_export_extensions/admin/model_admins/import_job_admin.py:41]: can we use tuple here?

File: import_export_extensions/admin/model_admins/import_job_admin.py [hunk 9]
```diff
@@ -259,5 +259,33 @@ def get_fieldsets(
 
         return [status, traceback_, import_params]
 
+    @admin.action(description="Cancel selected jobs")
+    def cancel_jobs(self, request, queryset):
```
Comment (yalef) [inline import_export_extensions/admin/model_admins/import_job_admin.py:263]: Let's add typings here too

File: README.rst [hunk 10]
```diff
@@ -43,14 +43,15 @@ To install `django-import-export-extensions`, run this command in your terminal:
 
     $ pip install django-import-export-extensions
 
-Add `import_export_extensions` to INSTALLED_APPS
+Add `import_export` and `import_export_extensions` to INSTALLED_APPS
```
Comment (yalef) [inline README.rst:46]: As I mention in other PR's I think we should use  double quotes instead of single ones

**Pull request merged**